src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+Unicode in regular expressions is well supported by modern browsers.
+
+{% Compare 'worse' %}
+```html
+<!-- Names with non-Latin characters (such as Françoise or Jörg) are 'invalid'. -->
+<input pattern="[\w \-]+" ...>
+```
+{% endCompare %}
+
+{% Compare 'better' %}
+```html
+<!-- Accepts Unicode letters. -->
+<input pattern="[\p{L} \-]+" ...>
+```
+{% endCompare %}
+
+
+<figure class="w-figure">
+   <video controls autoplay loop muted class="w-screenshot">
+     <source src="https://samdutton.com/unicode-letter-matching.mp4" type="video/mp4">
+   </video>
+  <figcaption class="w-figcaption">Unicode letter matching compared to Latin-only letter matching.</figcaption>
+</figure>
+
+### Allow for a variety of address formats {: #address-variety}
+
+When you're designing an address form, bear in mind the bewildering variety of address formats, 
+even within a single country. Be careful not to make assumptions about 'normal' addresses. (Take a 
+look at [UK Address Oddities!](http://www.paulplowman.com/stuff/uk-address-oddities/) if you're not 
+convinced!)
+
+#### Make address forms flexible {:flexible-address}
+
+Don't force users to [shoe-horn](https://en.wikipedia.org/wiki/Shoehorn#Turn_of_phrase) their address 
+into form fields that don't fit.
+
+For example, don't insist on a house number and street name in separate inputs, since many addresses 
+don't use that format, and incomplete data can break browser autofill.
+
+Be especially careful with `required` address fields. For example, addresses in large cities in the 
+UK do not have a county, but many sites still force users to enter one.
+
+Using two flexible address lines can work well enough for a variety of address formats.  
+
+```html
+<input autocomplete="address-line-1" id="address-line1" ...>
+<input autocomplete="address-line-2" id="address-line2" ...>
+```
+
+Add labels to match:
+
+```html/0-2,5-7
+<label for="address-line-1">
+Address line 1 (or company name)
+</label>
+<input autocomplete="address-line-1" id="address-line1" ...>
+
+<label for="address-line-2">
+Address line 2 (optional)
+</label>
+<input autocomplete="address-line-2" id="address-line2" ...>
+```
+You can try this out by remixing and editing the demo embedded below.
+
+{% Aside 'caution' %}
+Research shows that [**Address line 2** may be problematic for users]((https://baymard.com/blog/address-line-2)). 
+Bear this in mind when designing address forms—you should consider alternatives such as using a 
+single `textarea` (see below) or other options.
+{% endAside %}
+
+
+#### Consider using a single textarea for address {: #address-textarea}
+
+The most flexible option for addresses is to provide a single `textarea`.
+
+The textarea approach fits any address format, and it's great for cutting and pasting—but bear in 
+mind that it may not fit your data requirements, and users may miss out on autofill if they previously 
+only used forms with address-line1 and address-line2.
```

Comment:
```suggestion
The `textarea` approach fits any address format, and it's great for cutting and pasting—but bear in 
mind that it may not fit your data requirements, and users may miss out on autofill if they previously 
only used forms with `address-line1` and `address-line2`.
```